### PR TITLE
Theory Doc

### PR DESCRIPTION
**Pull request by sacha-escudier**
Started the documentation on the theory of FEM, and how we are implementing it in our project. I've also slightly updated mkdocs.yml to change citation styles on our website to render nicely, and to add a section on theory.

### CODE REVIEW

File: docs/assets/bibliography.bib [hunk 1]
```diff
@@ -7,3 +7,29 @@ @book{Serway2014
 	isbn = 9781133954057,
 	lccn = 2012947242
 }
+
+
+@misc{wiki:FEM,
```
[the remark this answers is not in the document]
Comment (sacha-escudier) [reply docs/assets/bibliography.bib]: I cited a reference on Comsol's website for now (not perfect), but I will fix it as I familiarize myself with more literature

**Comment by sinaatalay** [comment]
If your work is done, would you like to merge this as a first draft?

**Comment by sacha-escudier** [comment]
Hi guys,

I've just realized that I forgot to state it, but I should have made the appropriate changes. Could you review it and let me know if all looks good?

**Comment by sinaatalay** [comment]
@int-ptr-ptr Do you think it's ready to be merged?

File: docs/theory.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+-\int_\Omega  \mathbf{\nabla T} \cdot \mathbf{\nabla v} ds = \int_\Omega \frac{1}{\alpha} \frac{\partial T}{\partial t}ds + \int_\Omega g v ds
+$$
+
+Which may be simply reduced to a map $\phi$[@wiki:FEM]:
+
+$$
+-\phi(T,v) = \int_\Omega gv ds
+$$
+
+Where each function may be expressed as a basis of the shape functions $N_i(x,y)$ of the problem[@simulate2021fem], namely:
```
[the remark this answers is not in the document]
Comment (int-ptr-ptr) [reply docs/theory.md:53]: I'm resolving this for now, but we might want to revisit this in the future.

**Pull request merged**